pytest_mergify/resources/jenkins.py

```diff
@@ -1,9 +1,20 @@
+import os
+import typing
+
 from opentelemetry.sdk.resources import Resource, ResourceDetector
 from opentelemetry.semconv._incubating.attributes import cicd_attributes, vcs_attributes
 
 from pytest_mergify import utils
 
 
+def _get_branch() -> typing.Optional[str]:
+    branch = os.getenv("GIT_BRANCH")
+    if branch:
+        # remove remote
+        return branch.partition("/")[2]
```

Comment:
Not sure if the prefix is always here, in case not, this could be safer 
```suggestion
        return branch.split("/")[-1]
```
(otherwise it will return empty string with `partition`)

Reply:
@JulianMaurin, your version only returns the latest part of the `/`. 

```suggestion
        return branch..split("/", 1)[-1]
```
should be better?

Reply:
Let's me known WDYT